<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPage</class>
 <widget class="QMainWindow" name="TestPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="testpageLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>-10</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TEST PAGE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>60</y>
      <width>1281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Which play is this?</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="buttonGroup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string notr="true"/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="option1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Playname 1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="option2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Playname 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="option3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Playname 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="option4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Playname 4</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>791</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="playLayout"/>
   </widget>
   <widget class="QLabel" name="narrationLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>600</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
